Dynatrace/dynatrace-operator
# Enhance EdgeConnect CRD with additional connection properties

## Description

This PR introduces support for proxy and custom root certificates inside EdgeConnect CRD.

## How can this be tested?

Deploy the operator and apply EdgeConnect CR with new fields. Make sure new fields are projected inside EdgeConnect CR.

## Checklist
